command.js

```diff
@@ -41,16 +41,17 @@ DustJS.filters["convertType"] = function (value) {
         case 'double':
         case 'float':
         case 'int32':
-        case 'int64':
         case 'uint32':
-        case 'uint64':
         case 'sint32':
-        case 'sint64':
         case 'fixed32':
-        case 'fixed64':
         case 'sfixed32':
-        case 'sfixed64':
             return "number";
+        case 'int64':
+        case 'uint64':
+        case 'sint64':
+        case 'fixed64':
+        case 'sfixed64':
+			return "Long";
```

Comment:
Doesn't look like the tests were run for this change - this results in a `Cannot find name 'Long'. (TS2304)` error since "Long" is not a thing in typescript?